CONTRIBUTING.md

```diff
@@ -94,6 +94,19 @@ Here's what the contribution process looks like:
 
 > ⚛️ For RNW-specific walkthrough on setting up your branch, see the [Setup page](https://github.com/microsoft/react-native-windows/wiki/Setup) in our Wiki. 
 
+### 🔙 **Backporting**
```

Comment:
A gotcha is that changefiles (in the `/change` directory) may need to be changed from `prerelease` to `patch` is back ported from `main` to an already released branch. This is done automatically when backporting to 0.69+ though.

Reply:
@chiaramooney could you add this to the wiki page?